app/assets/javascripts/mobile.js

```diff
@@ -30,10 +31,18 @@ $(document).ready(function(){
 
   /* Drawer menu */
   $('#menu_badge').bind("tap click", function(evt){
```

Comment:
#5483 might be related to this issue: tapping twice would fire 2 events (to the user the first event wasn't recognized, because of the said delay). This leads to the "gost click" problem. 
http://stackoverflow.com/questions/20225153/preventing-ghost-click-when-binding-touchstart-and-click